save-agent/src/linuxX64Main/kotlin/org/cqfn/save/agent/SaveAgent.kt

```diff
@@ -213,7 +204,7 @@ class SaveAgent(private val config: AgentConfiguration,
      * @return a [HeartbeatResponse] from Orchestrator
      */
     internal suspend fun sendHeartbeat(executionProgress: ExecutionProgress): HeartbeatResponse {
-        // log.trace("Sending heartbeat to $orchestratorUrl")
+        logDebug("Sending heartbeat to ${config.orchestratorUrl}")
```

Comment:
Is this method still will return `HeartbeatResponse`

Reply:
Yes, it will try to deserialize server response as HeartbeatResponse and then use polymorphic serializer, that we have registered when creating httpClient

Reply:
Got it